docs/docs/introduction/oci-types.md

```diff
@@ -0,0 +1,149 @@
+the manifest endpoint:
+
+```
+GET /v2/<name>/manifests/<reference>
+```
+
+Do you want to upload a blob? There is an endpoint to POST to for that:
+
+```
+POST /v2/<name>/blobs/uploads/
+```
+
+**Where would I find a distribution specification?**
+
+An implementation of this specification would be found with a container registry that
+implements the OCI standard. For example, Docker Hub implements the specification. A
+developer could also roll their own registry (such as [this one](https://github.com/atlaskerr/stori))
+that implements it.
+
+**What would I do with the distribution specification?**
+
+The specification outlines a method of interaction. If you are writing a client, for example, these
+are the endpoints that the client would use. If you are creating a new registry that you wanted
+to conform to OCI, these are the API endpoints you would implement.
+
+**How might I contribute to the distribution specification?**
+
+Generally, since the goals are to move (push and pull) images, and provide endpoints for a client,
+if you are designing a registry or a client and find that you are missing some key function of an
+endpoint, or information provided by an endpoint, you might want to suggest a change to the specification.
+
+### The Runtime Specification
+
+The [runtime specification](https://github.com/opencontainers/runtime-spec/blob/master/spec.md) is another
+description of an interaction, but this time it's with a container binary.  Specifically, this means
+the execution environment, configuration, and lifecycle of a container.
+
+**What do you mean by execution environment?**
+
+What is the host operating system of your computer, Windows? Linux? These are different execution environments that come with different rules and standards for generating (what feels like) the same running container. If you look at the [runtime specification](https://github.com/opencontainers/runtime-spec/blob/master/spec.md) entry page, you'll actually find that there is a different specification file for each base operating system.
+
+**Why do we need runtime specifications?**
+
+As a user, you expect a container run on a Windows machine to generally feel like and act the same
+as a container run on a Linux host. The runtime specifications are catered toward this goal, with different
+configuration files (called config.json) to drive the creation, execution enviroinment, and actions 
+for the container.
+
+**How might I contribute?**
+
+You might have an entirely different operating system or technology that you want to explore running containers on, in which case you would want to contribute a new specification. You might find that there is an ambiguity or bug for one runtime, and ask a question about how to work on it.
+
+What are you waiting for? We want to hear from you!
+
+[Contribute]({{ site.baseurl }}docs/contribute/){: .btn .btn-primary .fs-5 .mb-4 .mb-md-0 .mr-2 }
```

Comment:
opencontainers/runc is another project, and possibly the most important of the OCI projects. It includes code used by most major container runtimes today to implement the OCI specifications.

Reply:
Added